### ISSUE TITLE
ERROR: Out of memory: failed to allocate 150994944 bytes on device 0 - in DIGITS 6

### ISSUE THREAD

**Issue by Tomue** [NVIDIA/DIGITS#1942]
I installed DIGITS 6 and wanted to train the same example I already trained on DIGITS 5 with the same GPU (1080Ti).

My actual setup:
DIGITS version:
6.1.0
Caffe version:
0.15.14
Caffe flavor:
NVIDIA

-- Dependencies:
--   BLAS              :   Yes (Atlas)
--   Boost             :   Yes (ver. 1.58)
--   glog              :   Yes
--   gflags            :   Yes
--   protobuf          :   Yes (ver. 2.6.1)
--   lmdb              :   Yes (ver. 0.9.17)
--   LevelDB           :   Yes (ver. 1.18)
--   Snappy            :   Yes (ver. 1.1.3)
--   OpenCV            :   Yes (ver. 2.4.9.1)
--   CUDA              :   Yes (ver. 8.0)
-- 
-- NVIDIA CUDA:
--   Target GPU(s)     :   Auto
--   GPU arch(s)       :   sm_61
--   cuDNN             :   Yes (ver. 5.1.10)
--   NCCL              :   Yes

I have no idea why this error occurs, I hope I missed something.

Thanks.

**Comment by ranju2015** [comment]
reduce learning rate and try (start with 0.001)

**Comment by Tomue** [comment]
Hi ranju

Thank you for your answer, but the batch size is 1.

**Comment by A1exy** [comment]
You can read tensorflow_output.log or caffe_output.log,  where contains details of the error.
